app/stac_api/collection_spatial_extent.py

```diff
@@ -87,19 +88,21 @@ def update_bbox_extent(self, trigger, geometry, original_geometry, item):
                         },
                     )
                     start = time.time()
-                    raw_sql = '''
-                    SELECT
-                    1 AS id,
-                    st_AsText(st_extent(geometry)) AS geometry__extent
-                    FROM stac_api_item
-                    WHERE collection_id = %d
-                    AND NOT id = %d
-                    ''' % (self.pk, item.pk)
-                    qs = type(item).objects.raw(raw_sql)
-                    union_geometry = GEOSGeometry(geometry)
-                    self.extent_geometry = Polygon.from_bbox(
-                        union_geometry.union(GEOSGeometry(qs[0].geometry__extent)).extent
-                    )
+                    qs_bbox_other_items = type(item).objects.filter(collection_id=self.pk
+                                                                   ).exclude(id=item.pk).aggregate(
+                                                                       Extent('geometry')
+                                                                   )
+                    geometry_updated_item = GEOSGeometry(geometry)
```

Comment:
Make sure to run yapf. It doesn't seem that yapf would leave such formatting but I might be wrong.